src/platform_linux/platform.h

```diff
@@ -669,6 +671,12 @@ platform_heap_create(platform_module_id module_id,
 void
 platform_heap_destroy(platform_heap_id *heap_id);
 
+void
+platform_shm_set_splinterdb_handle(platform_heap_id heap_id, void *addr);
+
+shmem_heap *
+platform_heap_id_to_shmaddr(platform_heap_id hid);
```

Comment:
Add extern references to these interfaces which are used [mainly] in unit-tests.